searx/templates/simple/search.html

```diff
@@ -1,6 +1,6 @@
 <form id="search" method="{{ method or 'POST' }}" action="{{ url_for('search') }}">
   <a id="search_logo" href="{{ url_for('index') }}">
-    <img class="search_logo_img" src="{{ url_for('static', filename='img/favicon.png') }}">
+    <img class="search_logo_img" src="{{ url_for('static', filename='img/searxng-wordmark.svg') }}">
```

Comment:
It can be replaced by an optimized inline version of the SVG file:
```svg
<svg viewBox="0 0 92 92" version="1.1"><g transform="translate(-40.92 -17.42)"><circle cx="75.92" cy="53.9" r="30" fill="none" stroke="#3050ff" stroke-width="10"/><path d="M67.51 37.92a18 18 0 0 1 21.06 3.3 18 18 0 0 1 3.13 21.09" fill="none" stroke="#3050ff" stroke-width="5"/><rect width="18.85" height="39.96" x="3.71" y="122.09" ry="0" transform="rotate(-46.23)" fill="#3050ff"/></g></svg>
```

This version be an external file that is included using the Jinja2 syntax.

The version above was made using https://jakearchibald.github.io/svgomg/ with these options:
* set precision to 2
* check "prefer viewbox to width/height" (at the bottom of the options)
* Manually remove `xmlns="http://www.w3.org/2000/svg"`.

For reference:
* https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_In_HTML_Introduction
* how the GitHub logo in the top left corner in embedded.

Reply:
> This version can be an external file that is included using the Jinja2 syntax.

I assume you are talking about something like this ... ?
```python

from flask import Markup
---
    return render(
        'results.html',
        wordmark_svg = Markup(open('.....img/searxng-wordmark.svg').read)
```
> To avoid an additional HTTP request

Yes, we can avoid one request, but on the opposite (if I'm not wrong) we lost flexibility with the `static/themes/simple` folder 

https://github.com/searxng/searxng/blob/580a55fe866b82e93dec6bc4e07fa8a0f2056475/searx/settings.yml#L64-L66

Reply:
What about 

webapp.py:
```python
for indice, theme in enumerate(themes):
    ...
    with os.path.join(settings['ui']['static_path'], 'theme', theme, 'img/favicon.png', 'rb') as f:
        favicon_content[theme] = f.read()
    ...


def render(...):
  ...
  kwargs['favicon_content'] = favicon_content[kwargs['theme']]
  ...


@app.route('/favicon.ico')
def favicon():
    return Response(favicon_content[get_current_theme_name()], mimetype='image/vnd.microsoft.icon')
```

search.html:
```jinja2
{{% favicon_content %}}
```

?

Reply:
@dalf I have tried what you suggested, see dd4da6f / but now I have a problem to size the inline SVG, see 
https://github.com/searxng/searxng/pull/473/commits/dd4da6f465349488ef101d76773d630dffea0d24#r743818450